templates/template_helpers.go

```diff
@@ -400,6 +402,20 @@ func slice(start interface{}, end interface{}, increment interface{}) ([]int, er
 	return out, nil
 }
 
+// Return the keys in the given map. This method always returns the keys in sorted order to provide a stable iteration
+// order.
+func keys(m map[string]string) []string {
```

Comment:
I added an extra helper here to make using Go's template engine more practical with maps. To be honest, I'm not too happy with Go templating. Like the rest of the Go language, it is missing a huge number of very basic features. Perhaps, in the future, we should move to a different templating engine.